### PR TITLE
zt/transactions

### PR DESCRIPTION
**Pull request by ztoolson**
Add Atrium::Transaction class to list and read transactions. Also introduce RSpec into the gem and add specs :)

//RFC @vintagepenguin @jjcarstens

### CODE REVIEW

File: lib/atrium/transaction.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    attribute :original_description
+    attribute :posted_at
+    attribute :status
+    attribute :top_level_category
+    attribute :transacted_at
+    attribute :type
+    attribute :updated_at
+    attribute :user_guid
+
+    def self.list(user_guid:)
```
Comment (jjcarstens) [inline lib/atrium/transaction.rb]: what does having `:` on the method argument do? (the `user_guid:`)

Comment (ztoolson) [inline lib/atrium/transaction.rb]: It is a feature in ruby 2 called keyword arguements. Here is an article on what it does, but basically you can call the method using a hash with the required arguements in any order.

`Atrium::Transaction.list(:user_guid => "USR-123")` is how you would call this.

https://robots.thoughtbot.com/ruby-2-keyword-arguments

Comment (jjcarstens) [inline lib/atrium/transaction.rb]: TIL

**Comment by ztoolson** [comment]
Part of https://github.com/mxenabled/atrium-ruby/issues/2

**Pull request merged**